presto-docs/src/main/sphinx/release/release-0.255.rst

```diff
@@ -0,0 +1,43 @@
+=============
+Release 0.255
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix an issue where regular expression functions were not interruptible and could  keep running for a long time after the query they were a part of failed.
```

Comment:
extra space between `could` and `keep` ?

Reply:
good catch